modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -0,0 +1,65 @@
+= Policy Level
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline, resource level
+
+By default, policies are applied to the entire API. However, you can implement an additional level of granularity for policies, where access is controlled based on the a criteria, called resource-level policies. 
+
+In this level of granularity, policies are applied to only those request that match the criteria.  All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
+
+//* SLA-based policies
+//Impose a limit specified by the level of access granted to the requesting application.The Rate Limiting policy is SLA-based.
+// Is SLA based a policy level granularity or a configuration level? because it doesn't change where the policy is applied, just the configuration ( a.k.a limit) - Eva??
+
+
+//[PA: We need to tell the customers how to work with SLA-tiers for policies right?]
+//SLA tiers for me should go somewhere else and link to all the documentation on contracts, request api access and those flows. - Eva?
+
+Resource level policy support includes RAML and HTTP-based APIs (since Mule 4). A policy applied at a resource level affects all HTTP methods, for example, PUT, GET, POST, or selected HTTP methods, within the resource.
+
+image::methods-resources.png[]
+
+You can apply multiple conditions to filter your resources and HTTP methods. The URI template regex can be applied to one, many, or all the methods that your API has.
+
+== Usage Scenarios
+
+The uses for resource level policies are limitless. Listed below are a few examples of how you might implement the resource-level policies:
+
+* Applying policies to specific resources
+* Securing a subset of an API
+* Setting different limits on resources
+
+If you have a RAML/OAS spec attached to your API, click Preview Resource Matching to check which resources your filters affect.
+
+Example API:
+
+image::users-api-cropped.png[API Console displays resources and methods,height=329,width=732]
+
+A resource level policy supports Java regular expressions. For example, you can use the wildcard to apply a policy to multiple resources. When you apply the policy to the API, specify the resources to which it applies.
+
+// were there differences between mule 3 and mule 4 in the expressions? if there are we should highlight them here -- Fede Balbi?
+
+image::rlp-conditions.png[]
+
+You can apply multiple conditions to filter your resources and methods. The URI template regex can be applied to one, many, or all the methods that your API has.
+
+Do _not_ use a placeholder, such as {userid}, in the regular expression. Using a placeholder in an expression fails because the placeholder does not match the actual node. In the case of the example placeholder {userid}, the node actually looks something like this:
```

Comment:
WHY are we illustrating what we do not want our users to do? That seems very dangerous, based on how careless readers can be. I suggest limiting the following text to the first sentence only, and omitting the rest, as well as the image.
```suggestion
Do not use a placeholder, such as {userid}, in the regular expression. 
```

Reply:
Removed some and kept some as I thought worked. Done! Thanks!